tests/metrics/test_inter_chunk_latency_metric.py

```
@@ -118,20 +118,7 @@ def test_inter_chunk_latency_invalid_order(self):
-
-    def test_inter_chunk_latency_equal_timestamps(self):
-        """Test error when consecutive responses have equal timestamps"""
-        record = create_record(
-            start_ns=100, responses=[110, 110, 120]
-        )  # Equal timestamps
-
-        metric = InterChunkLatencyMetric()
-        with pytest.raises(
-            ValueError,
-            match="Each inter chunk latency must be positive",
+            match="Each inter chunk latency must be non-negative. got: .*",
```

Ruff (0.14.0)

Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)